pkg/manifests/nginx.go

```diff
@@ -201,6 +219,10 @@ func newNginxIngressControllerService(conf *config.Config, ingressConfig *NginxI
 		annotations["external-dns.alpha.kubernetes.io/internal-hostname"] = "clusterip." + hostname
 	}
 
```

Comment:
Just want to call out that user will not be able to edit this resource in our new design, which means it will always be applied. I'm okay with this as nothing really changes but want to check with @sabbour on if adding these annotations should be optional?

Reply:
They can edit the resource still. Patch doesn't override all annotations (just the ones specified).